config/subsidy-application-management/forms/urban-renewal/application-phase2/versions/20221218090000/form.ttl

```diff
@@ -485,9 +490,464 @@ ext:ProjectfaseringMetProjectverloopTimingEnRisicobeheerProjectGroup
+          form:createGenerator ext:projectOnderdeelTimingListingGenerator;
+          form:canAdd true;
+          form:addLabel "Voeg stap toe";
+          form:canRemove true;
+          form:removeLabel "Verwijder";
+          form:tableIndexLabel "#";
+          form:showTableRowIndex true;
+          sh:group ext:projectOnderdeelTimingListingPg;
+          sh:maxCount 20;
+          sh:order 2 .
+
+        ##########################################################
+        #  Subform: projectOnderdeelTimingListingUnit
+        ##########################################################
+        ext:projectOnderdeelTimingListingSf a form:SubForm;
+          form:includes ext:projectOnderdeelTimingListingUnitBeschrijvingF;
+          form:includes ext:projectOnderdeelTimingListingUnitStartDatumF;
+          form:includes ext:projectOnderdeelTimingListingUnitEindDatumF;
+          form:hasFieldGroup ext:projectOnderdeelTimingListingPg.
+
+
+        ##########################################################
+        # Field: Projectonderdeel Timing Listing Beschrijving Input
+        ##########################################################
+        ext:projectOnderdeelTimingListingUnitBeschrijvingF
+            a                 form:Field ;
+            sh:name           "Beschrijf de verschillende stappen" ;
+            sh:order          1 ;
+            sh:path           lblodSubsidie:projectOnderdeelTimingListingUnitBeschrijving ;
+            form:validations
+                              [ a                 form:MaxLength ;
+                                form:grouping     form:MatchEvery ;
+                                form:max          "500" ;
+                                sh:resultMessage  "Max. karakters overschreden." ;
+                                sh:path            lblodSubsidie:projectOnderdeelTimingListingUnitBeschrijving
+                              ],
+                              [ a                 form:RequiredConstraint ;
+                                form:grouping     form:Bag ;
+                                sh:resultMessage  "Dit veld is verplicht." ;
+                                sh:path           lblodSubsidie:projectOnderdeelTimingListingUnitBeschrijving            
+                              ] ;
+            form:displayType  displayTypes:defaultInput ;
+            sh:group          ext:projectOnderdeelTimingListingPg .
+
+        ##########################################################
+        # Field: Projectonderdeel Timing Listing Startdatum Input
+        ##########################################################
+        ext:projectOnderdeelTimingListingUnitStartDatumF
+            a                 form:Field ;
+            sh:name           "Startdatum" ;
+            sh:order          2 ;
+            sh:path           lblodSubsidie:projectOnderdeelTimingListingUnitStartDatum ;
+            form:displayType  displayTypes:date ;
```

Comment:
It seems we still need to hide the label when the date field is rendered in a table. I'll try to get that fixed and released soon.